frontend/src/app/api/v1/chat/[courseId]/stream/route.ts

```diff
@@ -0,0 +1,27 @@
+import { NextRequest } from 'next/server'
+import { ChatService } from '@/client'
+
+export async function POST(
+  request: NextRequest,
+  { params }: { params: { courseId: string } }
+) {
+  const body = await request.json()
+  
+  try {
+    const response = await ChatService.postApiV1ChatStream({
+      path: { course_id: params.courseId },
+      body,
```

Comment:
Why not add the headers in the `ChatService.postApiV1ChatStream({})`?

Reply:
Adding the headers here doesn't mean it will be sent as part of the request, infact, I was supposed to remove this function as the call is being made on the client side, not server side, so this function is useless as it were.

Reply:
Even better